thirdparty/maven/zimbra-maven/debian/control

```diff
@@ -0,0 +1,14 @@
+Source: zimbra-maven
+Build-Depends: debhelper (>= 9), m4, dpkg-dev (>= 1.15.7), zimbra-openjdk
+Section: utils
+Priority: optional
+Maintainer: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Uploaders: Zimbra Packaging Services <packaging-devel@zimbra.com>
+Standards-Version: 3.9.5
+Homepage:  https://maven.apache.org/
+
+Package: zimbra-maven
+Priority: optional
+Architecture: any
+Depends: ${shlibs:Depends}, ${misc:Depends} zimbra-openjdk
```

Comment:
I believe you're missing a comma after Depends}

I.e., it should be Depends}, zimbra-openjdk